benchmark/sparse/linear_regression/results.md

```diff
@@ -0,0 +1,24 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```
+
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.0                                                       |
+| TensorFlow       | v1.10.0                                                      |
+| MXNet            | v1.2.1                                                       |
+
+
+### Results 
+| Instance Type | GPUs  | Batch Size  | MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
```

Comment:
We can mention here if users want to reproduce exact result, they can use fixed seed 7.  In general, we should observe consist/similar result with different random datasets
Or we can show an average of a few runs.

Reply:
Fixed